NETPROGRAM/BasicExercise/Exercise12.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+namespace NETPROGRAM.BasicExercise
+{
+    public class Exercise12
+    {
+  
+        public bool SymmetryNumber(int n)
+        {
+            var length = n.ToString().Length;
+            var sum = 0;
+            var i = 1;
+            var temp = n;
+
+            while(n != 0)
+            {
+                sum =10 * sum + (n % 10);
+                n /= 10;
+                i++;
```

Comment:
Xoá nốt biến i, đâu có dùng làm gì đâu. Tự anh để ý chút chứ 💃

Reply:
Do hôm qua cái bài 13, anh làm cả ngày, tìm cách làm ngắn gọn không được, sai miết, sau quay lại dùng vòng for. Đuối nên tối, sửa không coi lại nhiều.